js/components/brandlink/BrandLink.js

```diff
@@ -10,7 +7,6 @@
  * @returns {string} HTML or empty
  */
 
-
 export const Brandink = (data, parrentClassName) => {
   if (Object.keys(data).length !== 1) return '';
 
```

Comment:
There are problems below in line 13
const { url } = links;
You don't have links in this module